### ISSUE TITLE
[SUGGESTION] add sub menu-items should be made more intuitive

### ISSUE THREAD

**Issue by senlin** [voldemortensen/menu-customizer#79]
In my opinion it should be made a lot more clear how to add sub menu items. It has taken me well over 30 mins to figure out that the reorder menu actually has arrows to indent a menu item.

**Comment by celloexpressions** [comment]
Drag-and-drop, like the admin screen has, is being worked on in #4. Do you have any other suggestions? The only thing I can think of would be to adjust the label of the "Reorder" button to also reference submenus in some way, but I haven't been able to come up with anything.

**Comment by senlin** [comment]
One would expect it to work as it does now, so definitely without the need to click a button (reorder) to do it. 
If that action (having to click something to be able to make a sub structure) stays, then the label indeed should be changed. To what indeed is very difficult... Can't think of something off the top of my head, if something comes to me I'll update here!

**Comment by smutek** [comment]
Some suggestions for the label -
- Hierarchy
- Menu Structure
- Structure
- Grouping

**Comment by celloexpressions** [comment]
The button should definitely have a verb in its label regardless. We can't persistently show the reorder buttons/mode due to space, so I think improving that button is probably the best option here.

**Comment by senlin** [comment]
"Customize (incl. sub-menus)"

Regarding lack of space, that only is an issue, because you guys have made it an issue. Nobody dictates that the menu customizer cannot expand further into the screen. But of course that is a topic for a different discussion altogether... ;)

**Comment by celloexpressions** [comment]
We need to avoid using the term "customize" to refer to anything but the main Customizer panel/action within the Customizer. If we do change the label it needs to be one or possibly two words to be viable.

**Comment by celloexpressions** [comment]
"Sort" or "Sort & Group" or possibly "Restructure" might work. But I'm not a fan of any of those. Any ideas @designsimply?

**Comment by senlin** [comment]
None of the proposed words give the user the idea that submenu structure can only be applied through that button.
This simply will not work and this simply should not be how the whole thing works. With the current menu screen it is super simple *AND* super-intuitive to make a child, grand-child, etc. 
It simply is bizarre that you're just removing that functionality due to "lack of space". If that indeed is the reason then that is the biggest proof that adding it to the Customizer is a bad idea from the get-go.

**Comment by voldemortensen** [comment]
@senlin, we aren't removing any functionality. Once #4 is merged, the drag and drop functionality will exist just as it is now, as has been mentioned [here](https://github.com/voldemortensen/menu-customizer/issues/79#issuecomment-109490484). #4 is currently being worked on in the feature/sub-menus branch. Work is also track-able in #63.

**Comment by folletto** [comment]
_First of all: please always include a screenshot when discussing about or changing pieces of UI. It helps everyone to visualize what we are talking about in an unequivocal way._

—

On .com we conducted user tests on Menus and we discovered that drag'n'drop was actually used way more than expected (_91% more uses — last month's data_). So I agree with @voldemortensen that once #4 is merged we should be good. 

In pure terms of label, I think that "Reorder" works well. If not, I feel that just "Move" could be a good alternative. But I'd stick with "Reorder" personally, unless later user testing once drag'n'drop lands will tell us otherwise.

**Comment by smutek** [comment]
It makes sense to leave "reorder" as is until testing, but I did want to mention that I don't feel that "reorder" is the best choice. For me, "reorder" implies the functionality that is already present through drag and drop.

The critical function of reorder, currently (at-least to my mind), is to allow the user to create and adjust the depth of sub menu items. It's the only place where this can be done and it isn't easy to find. Based on this it seems that, at the least, there should be some sort of icon to support this, and better supporting text.

Here's a quick mockup that gives the button some button styling, changes the text to "Adjust Depth" and adds a right facing icon, to suggest placing sub menu items.

![adjust-depth](https://cloud.githubusercontent.com/assets/1855870/8043015/1ea1b6bc-0df1-11e5-8de6-ba78f844dcd6.jpg)

Just a thought!

**Comment by senlin** [comment]
+1 to what @smutek says

**Comment by designsimply** [comment]
@senlin's suggestion to make sub-menus more intuitive by letting people drag & drop to create them is an excellent suggestion. I think that's the answer.

Closing this as a duplicate of #4 where dragging to reorder items into submenus is being worked on, and we will re-open this issue if people run into trouble with the 'reorder' label in user testing.
